### DISCUSSION TITLE
List or email LNURL Comments

### DISCUSSION THREAD

**Discussion by Sjors** [btcpayserver/btcpayserver#4739] [Ideas / Feature Requests]
It would be nice to have an overview of LNURL comments (and similar notes that go along with a lightning payment). Right now I tend to miss them, because you have to open a payment and look for it.

Could be simple table: date, amount, comment

Alternatively / additionally, it would be nice to receive emails for them. Though that could get noisy (with zaps and boostagrams), so perhaps they should be batched per day / week.

**Comment by dennisreimann** [comment]
Hey @Sjors, see what I proposed here https://github.com/btcpayserver/btcpayserver/pull/4422
